tests/core/test_typing.py

```diff
@@ -292,3 +293,33 @@ def test_conform_primitives():
     assert _conform_primitive_property(None, {"type": "boolean"}) is None
     assert _conform_primitive_property(0, {"type": "boolean"}) is False
     assert _conform_primitive_property(1, {"type": "boolean"}) is True
+
+
+import pytest
+import sqlalchemy
+
```

Comment:
@pnadolny13 can you move these imports to the top of the file?

Reply:
@edgarrmondragon yep sorry I forgot to clean this up. Fixing now